src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.WebSearch/BrowserInfo.cs

```diff
@@ -0,0 +1,205 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.IO.Abstractions;
+using System.Text;
+using Wox.Plugin.Logger;
+
+namespace Community.PowerToys.Run.Plugin.WebSearch
+{
+    public class BrowserInfo
+    {
+        public const string MSEdgePath = @"C:\Program Files (x86)\Microsoft\Edge\Application\msedge.exe";
```

Comment:
This shouldn't be hardcoded.

Reply:
Don't agree. The path we use as falback should be hardcoded.

But I think the first part (program files) should be an environment variable.

Reply:
@htcfreek, yeah, I do mean the program files part of the path.

Reply:
And I personally would rename the vars to some more general like `fallbackBrowserPath` and `fallbackBrowserName`.